src/backend/distributed/utils/background_jobs.c

```diff
@@ -125,7 +125,7 @@ static volatile sig_atomic_t GotSighup = false;
 
 /* keeping track of parallel background tasks per node */
 HTAB *ParallelTasksPerNode = NULL;
-int MaxBackgroundTaskExecutorsPerNode = 1;
+int MaxBackgroundTaskExecutorsPerNode = 4;
```

Comment:
What's the implication of this? Only rebalance operation use these extra tasks or generic tasks too?

Reply:
If we don't launch them by default, limiting to 4 should fine.